## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-erasure.sh | :x: [more...](#db431781-da81-4a15-8ee1-60fc3389e2ae) |
<a name="db431781-da81-4a15-8ee1-60fc3389e2ae"></a>
### 12615-347c20c/mint-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s2:31216
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/16) Running aws-sdk-go tests ... done in 1 seconds
(2/16) Running aws-sdk-java tests ... done in 1 seconds
(3/16) Running aws-sdk-php tests ... done in 43 seconds
(4/16) Running aws-sdk-ruby tests ... done in 2 seconds
(5/16) Running awscli tests ... done in 59 seconds
(6/16) Running healthcheck tests ... done in 0 seconds
(7/16) Running mc tests ... done in 21 seconds
(8/16) Running minio-dotnet tests ... done in 26 seconds
(9/16) Running minio-go tests ... done in 1 minutes and 54 seconds
(10/16) Running minio-java tests ... FAILED in 11 seconds
Exception in thread "main" java.lang.AssertionError: object count; expected=1050, got=1049 expected:<1049> but was:<1050>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at FunctionalTest.testListObjects(FunctionalTest.java:1402)
	at FunctionalTest.listObjects(FunctionalTest.java:1447)
	at FunctionalTest.runObjectTests(FunctionalTest.java:3684)
	at FunctionalTest.runTests(FunctionalTest.java:3720)
	at FunctionalTest.main(FunctionalTest.java:3864)
(10/16) Running minio-js tests ... done in 48 seconds
(11/16) Running minio-py tests ... done in 58 seconds
(12/16) Running s3cmd tests ... done in 24 seconds
(13/16) Running s3select tests ... done in 1 seconds
(14/16) Running security tests ... done in 0 seconds
(15/16) Running versioning tests ... done in 3 minutes and 2 seconds

Executed 15 out of 16 tests successfully.
```
Deleting image on docker hub
Deleting image locally

--- a/cmd/storage-rest-server.go
+++ b/cmd/storage-rest-server.go
@@ -212,6 +212,10 @@ func (s *storageRESTServer) NSScannerHandler(w http.ResponseWriter, r *http.Requ
 	if err = respW.WriteBool(false); err == nil {
 		err = usageInfo.EncodeMsg(respW)
 	}
+	if err != nil {
+		resp.CloseWithError(err)
+		return
+	}
 	resp.CloseWithError(respW.Flush())
 }
 

